app/services/moderator/audit/notification.rb

```diff
@@ -0,0 +1,24 @@
+module Moderator
```

Comment:
What do you think about using `Audit`(or similar) as a top-level namespace? We may want to have other auditing features in the future and it would be nice to have them in one place. `Moderator` actions classes that are already in this namespace seem to be less related to this feature.